assets/src/components/d3/createResourceAccessChart.js

```diff
@@ -393,15 +393,34 @@ function createResourceAccessChart ({ data, width, height, domElement }) {
     .on('mousedown.brush', brushcenter)
     .on('touchstart.brush', brushcenter, { passive: true })
 
+  let defs = svg.append('defs');
```

Comment:
I see red Squiggly line near this. Myla adapted the Standard based linter https://standardjs.com/. There is a VSCode Plugin for it. I configured VCode to  `ts-standard`. There 2 places in the code needs a change
![Screenshot 2023-11-21 at 11 05 59 AM](https://github.com/tl-its-umich-edu/my-learning-analytics/assets/8579775/0113e318-0313-4600-8a80-cde98f805590)

![Screenshot 2023-11-21 at 11 06 13 AM](https://github.com/tl-its-umich-edu/my-learning-analytics/assets/8579775/b6910200-6eb9-4d94-b210-9699e868914c)

![Screenshot 2023-11-21 at 11 06 17 AM](https://github.com/tl-its-umich-edu/my-learning-analytics/assets/8579775/5ddca86f-1ce1-4b94-a6f0-ea02d38c6f76)

install the linter and make the change

Reply:
gotcha, added standard and fixed some of these issues it was pointing out. Thanks!